lib/bemhtml/runtime/match.js

```diff
@@ -166,6 +166,14 @@ Match.prototype.push = function push(template) {
   this.maskSize = this.mask.length;
 };
 
+Match.prototype.tryCatch = function(fn, ctx) {
+  try {
+    return fn.call(ctx);
+  } catch (e) {
+    return e;
```

Comment:
Let's create a property called `.thrownError` in constructor, and initialize it to `null`.